modules/examples/eureka/manifests/fsx_static_provisioning.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: fsx-sc
+provisioner: fsx.csi.aws.com
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: fsx-pv
+spec:
+  storageClassName: fsx-sc

```

Comment:
pls update the PV object from the example here https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/examples/kubernetes/static_provisioning/README.md

Reply:
Removing storage class as well.